Updates to readme file. (#124)
wickedest/Mergely: README.md

http://mergely.com

> Mergely is a javascript component to differencing and merging files interactively in a browser, providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  

Reviewer: There is something funky with the verb "to differencing and merging", and it also loses the technical jargon "diff", which is important to retain.

```suggestion
Mergely is a JavaScript component for differencing and merging files interactively in a browser (diff/merge), providing rich API that enables you to easily integrate Mergely into your existing web application. It is suitable for comparing text files online, for example, .txt, .html, .xml, .c, .cpp, .java, etc.  
```

Author: Thanks for the tech explanation :)

> Mergely has a javascript implementation of the Longest Common Subsequence (LCS) diff algorithm, and a customizable markup engine.

Reviewer: ```suggestion
Mergely has a JavaScript implementation of the Longest Common Subsequence (LCS) diff algorithm, and a customizable markup engine.
```

## Installation

### Installation via webpack

> The recommended way to install mergely and its dependencies is to use a Node package manager (npm) and [webpack](https://webpack.js.org/).

Reviewer 2: Just a small suggestion:

```suggestion
The recommended way to install mergely and its dependencies is to use a Node package manager (`npm` or `yarn`) and [webpack](https://webpack.js.org/).
```

It is highly recommended that you start by cloning [mergely-webpack](https://github.com/wickedest/mergely-webpack). The webpack has everything that you need to get started.

### Angular 6.1.1
You can also use mergely within angular. You can start by cloning [mergely-angular](https://github.com/wickedest/mergely-angular).

### Installation via .tgz

> Unpack mergely.tgz into a folder, for example, `./lib`, and add the following to the `<head>` of your target HTML source file.

Reviewer 2: Minor nit: There should be no comma after "for example".

> Unpack mergely.tgz into a folder, for example, `./lib`, and add the following to the `<head>` of your target HTML source file.

Reviewer: ```suggestion
Unpack mergely.tgz into a folder, for example `./lib`, and add the following to the `<head>` of your target HTML source file.
```

Author: https://www.thefreedictionary.com/for+example

Reviewer 2: None of those examples apply to this case, IIUC. But that's a minor nit, really, so no need to bikeshed it :)

### Synchronous content initialization

> The following example can be used to set the `lhs` and `rhs` editors synchronously, that is, when their contents are already known):

Reviewer 2: You removed the opening parenthesis, so the closing one is unbalanced. It should be restored:

```suggestion
The following example can be used to set the `lhs` and `rhs` editors synchronously (that is, when their contents are already known):
```

### resize

> Resize the editor. It must be called explicitly if `autoresize` is disabled.

Reviewer 2: ```suggestion
Resizes the editor. It must be called explicitly if `autoresize` is disabled.
```

Note that the `### search` section below also needs to be adjusted to match verbal tense ("Searches...")

### update

> Manual update, recalculates the diff and applies new settings.

Reviewer 2: The `;` to `,` is not appropriate — it introduces a run-on sentence.

Maybe this could be reworked into, e.g.

> Manually updates the editor by recalculating the diff and applying new settings

WDYT?

> Manual update, recalculates the diff and applies new settings.

Reviewer: ```suggestion
Manually updates the editor by recalculating the diff and applying new settings.
```
